src/device-state.coffee

```diff
@@ -244,7 +244,7 @@ module.exports = class DeviceState extends EventEmitter
 			@applications.services.killAllLegacy()
 		.then =>
 			log.debug('Migrating legacy app volumes')
-			@applications.getTargetApps()
+			@applications.getTargetApps({ availableImages: [] })
```

Comment:
nit: It might be nicer to make `availableImages` default to `[]` inside the function?